runtime/resolvers/metrics_sql.go

```diff
@@ -41,8 +39,8 @@ func newMetricsSQL(ctx context.Context, opts *runtime.ResolverOptions) (runtime.
 		return nil, err
 	}
 
-	var finalRefs []*runtimev1.ResourceName
-	props.SQL, finalRefs, err = resolveTemplate(props.SQL, opts.Args, instance, opts.Claims.UserAttributes, opts.ForExport)
+	// todo handle refs
+	props.SQL, _, err = resolveTemplate(props.SQL, opts.Args, instance, opts.Claims.UserAttributes, opts.ForExport)
```

Comment:
I think we need to handle that to ensure correct cache invalidation when refs are updated?

Reply:
So WRT `metrics_sql` the only ref as of now is `metrics_view` which is already handled in `metricsResolver` so it should be okay to ignore this for now ?

Reply:
Oh yeah true, should be fine then